### PR TITLE
all main tasks completed, except task of higher diffuclty

### PR DESCRIPTION
**Pull request by hereticant**
1. integration with VK
2. one to one connection with possibility to change data
3. automaticated procedure of fill via VK profile
4  AuthForbidden work checked, well worked
exception:
5. save native langauge and url link to profile

### CODE REVIEW

File: authapp/pipeline.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    if backend.name != 'vk-oauth2':
+        return
+
+    api_url = urlunparse(('https',
+                          'api.vk.com',
+                          '/method/users.get',
+                          None,
+                          urlencode(OrderedDict(fields=','.join(('bdate', 'sex', 'about')),
+                                                access_token=response['access_token'],
+                                                v='5.103')),
```
Comment (Roxe322) [inline authapp/pipeline.py:22]: А почему 5.103?

File: geekshop/settings.py [hunk 2]
```diff
@@ -22,9 +22,11 @@
 SECRET_KEY = '@vy=p8)kpo+!x)kx5*(vvv8s!a#0umsxt+907tul+3pt(gxtrz'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
+
+git
```
Comment (Roxe322) [inline geekshop/settings.py]: это что?)

File: authapp/pipeline.py (hunk 1 above)
Comment (hereticant) [reply authapp/pipeline.py:22]: то была последняя версия в ВК

File: geekshop/settings.py (hunk 2 above)
Comment (hereticant) [reply geekshop/settings.py]: ошибочка вышла)

**Pull request merged**